lisa/sut_orchestrator/azure/platform_.py

```diff
@@ -465,14 +467,20 @@ def _deploy_environment(self, environment: Environment, log: Logger) -> None:
         else:
             try:
                 if self._azure_runbook.deploy:
+                    if self._azure_runbook.resource_group_location:
```

Comment:
The env resource group can be the same as first node.

Reply:
fixed